pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -553,6 +553,12 @@ func Convert_v1_VirtualMachine_To_api_Domain(vmi *v1.VirtualMachineInstance, dom
 
 	networks := map[string]*v1.Network{}
 	for _, network := range vmi.Spec.Networks {
+		if network.Multus == nil && network.Pod == nil {
+			return fmt.Errorf("fail network %s must have a network type", network.Name)
+		}
+		if network.Multus == nil && network.Pod == nil {
```

Comment:
This will never match because we return in line 557. I believe you mean:

```
if network.Multus != nil && network.Pod != nil {
...
}
```

Reply:
Yes bad copy paste. 
Thanks!